sysl2/grammarParser.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+    sysl "anz-bank/sysl/sysl2/proto"
+    "sort"
```

Comment:
Move "sort" first, with a space between the two.

Reply:
vscode did its auto thing here again..
forcibly changed the order now.